### PR TITLE
Create Intelligence Documentation

### LINKED ISSUE #1981: Write documentation for the intelligence tab
With https://github.com/google/timesketch/pull/1968/ intel was added as a new feature.

Documentation is still open for that feature.

### PR DESCRIPTION
**Pull request by jaegeral**
Adding first stab at Intelligence Documentation

**Checks**

- [X] Documentation updated.

**Closing issues**

closes #1981

### CODE REVIEW

**Comment by tomchop** [comment]
@googlebot I consent.

File: docs/user-guide/intelligence.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Intelligence
+
+Each sketch can have Intelligence items associated with it. This comes in the form of simple strings extracted from events
```
Comment (itsmvd) [inline docs/user-guide/intelligence.md:3]: comma after events (extracted from events, usually as IOCs.)

File: docs/user-guide/intelligence.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+usually as IOCs. (e.g. a suspicious IP address in a log file that you want to highlight.)
+
+## Addding new intelligence
+
+Timesketch will parse events attributes for potential IOCs to be added as Intelligence. (e.g. when Timesketch finds something that looks like a sha256 hash, IP address, etc.) This will be surfaced to you by a grey highlight on the corresponding string. You can then click the highlighted string to open and pre-fill the `add local intelligence` dialog. You can optionally change the IOC type before confirming the suggestion.
```
Comment (itsmvd) [inline docs/user-guide/intelligence.md:8]: Dot (.) after intelligence should be after the parentheses that follow.

**Review by itsmvd — APPROVED** [review]
Smallest nits ever, LGTM

**Comment by tomchop** [comment]
Fixed @itsmvd's comments. @jaegeral can you resolve the conflicts and merge this?

**Pull request merged**